InternalPythonModules/android/calllog.py

```diff
@@ -95,11 +95,12 @@ def analyze(self, dataSource, fileManager, context):
                     ContentUtils.writeToFile(abstractFile, file, context.dataSourceIngestIsCancelled)
                     self.__findCallLogsInDB(file.toString(), abstractFile, dataSource)
                 except IOException as ex:
-                    self._logger.log(Level.SEVERE, "Error writing temporary call log db to disk", ex)
-                    self._logger.log(Level.SEVERE, traceback.format_exc())
+                    # Error writing temporary call log db to disk.
```

Comment:
Should we swallow this exception? I at least don't in my file selector module. This could lead to nasty debugging in the future.